kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -236,6 +238,10 @@ private Object convertLogical(Object kafkaConnectObject,
                                 Schema kafkaConnectSchema) {
     LogicalTypeConverter converter =
         LogicalConverterRegistry.getConverter(kafkaConnectSchema.name());
+
+    if(shouldConvertDebeziumTimestampToInteger && converter instanceof DebeziumLogicalConverters.TimestampConverter) {
+      return (Long) kafkaConnectObject;
```

Comment:
is the kafka schema type of this object always INT64 ?

Reply:
As per the [current](https://github.com/confluentinc/kafka-connect-bigquery/blob/39a43f9295d12b543e6a3ebe8f49c80e562e7087/kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/logicaltype/DebeziumLogicalConverters.java#L175) conversion of Timestamp object to String, it assumes the object to be of INT64 only